src/custom/utils/operator.ts

```diff
@@ -188,6 +188,15 @@ export type FeeQuoteParams = Pick<OrderMetaData, 'sellToken' | 'buyToken' | 'kin
   chainId: ChainId
 }
 
+function toApiAddress(address: string, chainId: ChainId): string {
+  if (address === 'ETH') {
+    // TODO: Return magical address
+    return WETH[chainId].address
+  }
```

Comment:
What about xDai?

Reply:
This is a fix for xDAI :) for rinkeby/mainnet was working, cause they were accepting ETH before. 
They want for now WETH address, and later the eEeEeEEeEEEeeeeee address